server/embedly.js

```diff
@@ -0,0 +1,109 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+// Facade that abstracts the embed.ly endpoint.
+// TODO use Joi to validate the url and ensure key is defined
+
+var camelize = require('underscore.string').camelize;
+var Embedly = require('embedly');
+
+var config = require('./config');
+var apiKey = config.get('embedly.apiKey');
+var isEnabled = config.get('embedly.enabled');
+var logFactory = require('./logger');
+var log = logFactory('server.services.embedly');
+var npmLog = logFactory('server.services.embedly.vendor');
+
+module.exports = {
+  extract: function extract(url, cb) {
+    log.debug('embedly.extract called');
+
+    if (!isEnabled) {
+      return cb('Embedly is currently disabled. Set `server.embedly.enabled` to enable.');
+    }
```

Comment:
Probably should also include a helpful error if the apiKey is undefined, since that seems like an easy thing to forget.